notebooks/official/generative_ai/rlhf_tune_llm.ipynb

```diff
@@ -847,12 +835,12 @@
       },
       "source": [
         "## Get Predictions from Tuned Models\n",
-        "The recommened way to get predictions from tuned models depends on whether you are working with a first-party or third-party model. For first-party models, e.g. `text-bison@001`, you can get predictions using [Batch Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-batch-predictions) or [Online Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-online-predictions), and for third-party models, we provide a bulk inference pipeline that can be used with the tuned-model checkpoint. Examples for both model types are in subsequent sections below.\n",
+        "The recommened way to get predictions from tuned models depends on whether you are working with a first-party or third-party model. For first-party models, e.g. `text-bison@001`, you can get predictions using [Batch Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-batch-predictions) or [Online Prediction](https://cloud.google.com/vertex-ai/docs/predictions/get-online-predictions), and for third-party models, a bulk inference pipeline is provided that can be used with the tuned-model checkpoint. Examples for both model types are in subsequent sections below.\n",
         "\n",
         "### Bulk Inference for Third-Party Models\n",
-        "In this section we show how to use the bulk inference pipeline to get predictions from a tuned `t5` or `llama2` model. This method will not work for first-party models, e.g. `text-bison@001`. Follow the example in the next section to get predictions from those models.\n",
+        "In this section you learn how to use the bulk inference pipeline to get predictions from a tuned `t5` or `llama2` model. This method doesn't work for first-party models, e.g. `text-bison@001`. Follow the example in the next section to get predictions from those models.\n",
         "\n",
-        "To perform bulk inference you will need the path to your tuned third-party model, which can be found in the Vertex Pipelines UI under **Reinforcer** > **Output Parameters** > `output_model_path`. Once you have the model path, follow the directions in the [Bulk Inference notebook](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/batch_eval_llm.ipynb) to generate offline predictions from a tuned model checkpoint."
+        "To perform bulk inference you need the path to your tuned third-party model, which can be found in the Vertex Pipelines UI under **Reinforcer** > **Output Parameters** > `output_model_path`. Once you have the model path, follow the directions in the [Bulk Inference notebook](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/generative_ai/batch_eval_llm.ipynb) to generate offline predictions from a tuned model checkpoint."
```

Comment:
- add comma: " "To perform bulk inference, you need the path..."

- change: "Vertex Pipelines UI"
- to: "Google Cloud console for Vertex AI Pipelines"